03 - Data Science and Custom Skills/Azure Function Custom Skills/README.md

```diff
@@ -0,0 +1,4 @@
+# Custom skills have moved!
```

Comment:
Can you add a bit more information on what Power Skills are... Maybe sell it a bit more, instead of just telling people that they moved? :-)